src/render/OpenGL.cpp

```diff
@@ -1597,6 +1605,16 @@ void CHyprOpenGLImpl::renderTextureInternalWithDamage(SP<CTexture> tex, const CB
         shader->setUniformFloat2(SHADER_FULL_SIZE, m_renderData.pMonitor->m_pixelSize.x, m_renderData.pMonitor->m_pixelSize.y);
     }
 
+    if (usingFinalShader && *PDT == 0) {
+        shader->setUniformFloat2(
+            SHADER_POINTER,
+            g_pInputManager->getMouseCoordsInternal().x / m_renderData.pMonitor->m_pixelSize.x,
+            g_pInputManager->getMouseCoordsInternal().y / m_renderData.pMonitor->m_pixelSize.y
```

Comment:
nuh uh, mouse coords are in logical space. Gl vec would be:
```cpp
g_pInputManager->getMouseCoordsInternal().translate(-pMonitor->vecPosition).scale(pMonitor->scale).transform(pMonitor->transform, pMonitor->vecTransformedSize.x, pMonitor->vecTransformedSize.y);
```
With the transform order I am not sure. Please check it.